code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -68,7 +111,16 @@
 	wrapped = null
 	//update_icon()
 
-/obj/item/weapon/gripper/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob)
+/obj/item/weapon/gripper/attack(mob/living/carbon/C as mob)
+	//Medical Grippers can shape people awake
+	if(istype(src,/obj/item/weapon/gripper/medical))
```

Comment:
Actually IIRC the arguments get passed down from the parent, but it's not something you should do either way.